apps/project/schema.py

```diff
@@ -581,6 +618,20 @@ class Meta:
         filterset_class = PublicProjectByRegionGqlFileterSet
 
 
+class ProjectSummaryStatEntryActivityType(graphene.ObjectType):
+    project_id = graphene.ID()
+    count = graphene.Int()
+    date = graphene.Date()
+
+
+class ProjectSummaryStatType(graphene.ObjectType):
+    projects_count = graphene.Int()
+    total_leads_count = graphene.Int()
+    total_leads_tagged_count = graphene.Int()
+    total_leads_tagged_and_controlled_count = graphene.Int()
+    recent_entries_activities = graphene.List(graphene.NonNull(ProjectSummaryStatEntryActivityType))
```

Comment:
```suggestion
    projects_count = graphene.Int(required=True)
    total_leads_count = graphene.Int(required=True)
    total_leads_tagged_count = graphene.Int(required=True)
    total_leads_tagged_and_controlled_count = graphene.Int(required=True)
    
    recent_entries_activities = graphene.List(
        graphene.NonNull(ProjectSummaryStatEntryActivityType),
        required=True,
    )
```

Reply:
![image](https://github.com/the-deep/server/assets/7059255/ddb696c0-a8f4-4556-aeb4-61b25b08906f)
```
    recent_entries_activities = graphene.List(
        graphene.NonNull(ProjectSummaryStatEntryActivityType),
        required=True,
    )
```